cfme/fixtures/pytest_selenium.py

```diff
@@ -319,6 +326,24 @@ def _nothing_in_flight():
         num_sec=_thread_local.ajax_timeout, delay=0.1, message="wait for ajax", quiet=True,
         silent_failure=True)
 
+    # If we are not supposed to take page screenshots...well...then...dont.
+    if not store.config.getvalue('page_screenshots'):
+        return
+
+    url = browser().current_url
+    url = url.replace(base_url(), '')
+    url = url.replace("/", '_')
+    if url not in urls:
+        logger.info('Taking picture of page: {}'.format(url))
+        ss, sse = take_screenshot()
+        if ss:
+            ss_path = log_path.join('page_screenshots')
+            if not os.path.exists(ss_path.strpath):
+                os.mkdir(ss_path.strpath)
```

Comment:
Why you use `os` features when you can just do `ss_path.exists()` and ``ss_path.mkdir()`` ?